src/components/UI/ButtonStyled/RadioButtonstyled.js

```diff
@@ -4,16 +4,9 @@ export const WrapperFieldset = styled.div`
 	display: grid;
 	grid-template-rows: repeat(1, 1fr);
 	justify-content: center;
-	width: 80vw;
+	width: 100vw;
```

Comment:
Why is your width set to 100vw? Here should 100% be do the same.

Reply:
Fixed